core/tests/recovery-test/src/index.ts

```diff
@@ -145,12 +147,49 @@ export class NodeProcess {
         }
     }
 
+    async stop(signal: 'INT' | 'KILL' = 'INT') {
+        interface ChildProcessError extends Error {
+            readonly code: number | null;
+        }
+
+        let signalNumber;
+        if (signal == 'KILL') {
+            signalNumber = 9;
+        } else {
+            signalNumber = 15;
+        }
+        try {
+            let childs = [this.childProcess.pid];
+            while (true) {
+                try {
+                    let child = childs.at(-1);
+                    childs.push(+(await promisify(exec)(`pgrep -P ${child}`)).stdout);
+                } catch (e) {
+                    break;
+                }
+            }
+            // We always run the test using additional tools, that means we have to kill not the main process, but the child process
+            for (let i = childs.length - 1; i >= 0; i--) {
+                console.log(`kill ${childs[i]}`);
```

Comment:
redundant log

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/2594/commits/323206e2c6e39b027c6bbb31d9a2015f796b104c